### PR TITLE
[feat] added Homepage and Explore pages

### PR DESCRIPTION
**Pull request by omjadhav85**
1. Added Homepage and Explore pages
2. Added api calls to get all videos from database
3. Minor ui corrections/changes

### CODE REVIEW

File: src/components/VideoCard/VideoCard.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.video-card {
+  display: flex;
+  justify-content: center;
+}
+
+.video-card .card {
+  width: 320px;
+  height: 256px;
+  max-height: 100%;
```
Comment (Kothalkarpriya) [inline src/components/VideoCard/VideoCard.css:9]: You should use rem for more than 10px sizing

File: src/components/VideoCard/VideoCard.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.video-card .card .card-body:hover {
+  background-color: transparent;
+  cursor: default;
+}
+
+.video-card .card .card-img:hover {
+  cursor: pointer;
+  opacity: 60%;
```
Comment (Kothalkarpriya) [inline src/components/VideoCard/VideoCard.css]: opacity can be used in the range of 0-1

File: src/styles/index.css [hunk 3]
```diff
@@ -13,7 +13,7 @@
 body {
   background-color: var(--background-color);
   font-family: "Comfortaa", cursive;
-  min-width: 300px;
+  min-width: 320px;
```
Comment (Kothalkarpriya) [inline src/styles/index.css]: for more than 10px you should use rem convention

File: src/utils/videoServerCalls.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+export const getAllVideos = async (setVideos) => {
+  try {
+    const res = await axios.get("/api/videos");
```
Comment (Kothalkarpriya) [inline src/utils/videoServerCalls.js:5]: Good that you used async await here

**Review by Kothalkarpriya — APPROVED** [review]
You can merge after looking into my suggestions

**Review by AnuragSahu11 — APPROVED** [review]
It's good that you have used catch statement for error handling, you can merge

**Comment by omjadhav85** [comment]
Thanks Priya and Anurag for your review and suggestions. Made the required changes. Merging this pr.

**Pull request merged**